### ISSUE TITLE
This library should be added to our docs

### ISSUE THREAD

**Issue by wlan0** [koki/templates#5]
the doc website is in koki/koki repository

**Comment by ublubu** [comment]
Good idea. The functionality here was actually pasted into `koki/short` and improved upon. At some point i should merge those improvements back into this library and consider importing this library into `koki/short`.

**Comment by wlan0** [comment]
Yes pls
